app/views/catalog/_osd_or_embed_default.html.erb

```diff
@@ -1,5 +1,8 @@
+<% @viewer = Viewer.find_by(exhibit: @exhibit) %>
```

Comment:
Can we use `viewer` instead then?  `@viewer` would indicate to me that we're setting a variable that is intended to have a greater scope than just this partial, where `viewer` would indicate that the scope is just to this partial (or wherever passed explicitly as a parameter or local).